extensions/widgets/navbar/navbar.lcb

```diff
@@ -1212,6 +1361,15 @@ private handler setNavData(in pNavData as Array)
 	end if
 end handler
 
+private handler setWidgetTheme(in pWidgetTheme as String)
+	if pWidgetTheme is mWidgetTheme then
+		return
+	end if
+
+	put pWidgetTheme into mWidgetTheme
+	redraw all
+end handler
+
```

Comment:
Since widgetTheme can be set from script, it is not insulated against users setting it to an invalid string value (we will be able to sort this out in a general way once we have proper enum types)

For now we've been dealing with it by setting up some constants and throwing an error if it doesn't match any of them. See the treeview's sortOrder and sortType properties for examples: https://github.com/livecodeali/livecode/commit/3671f6fb4ea30e310620eef3663a5657bab14dd4